auto_round/special_model_handler.py

```diff
@@ -28,6 +28,19 @@
     "idefics3"
 ]
 
+def _get_deepseek_vl2_multimodal_block(model, quant_vision=False):
+    model.forward = model.language.forward
```

Comment:
the origin model only quantize language part,  why this function still throw exception